src/index.html

```diff
@@ -3,11 +3,39 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product" data-qa="card">
+      <div class="product__photo"></div>
+      <h1 class="product__title"> APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</h1>
+      <p class="product__code">Product code: 195434</p>
+      <div class="product__rank">
+        <div class="product__rank-stars">
+          <div class="product__rank-stars-item"></div>
+          <div class="product__rank-stars-item"></div>
+          <div class="product__rank-stars-item"></div>
+          <div class="product__rank-stars-item"></div>
+          <div class="product__rank-stars-item"></div>
+        </div>
+        <p class="product__rank-reviews"> Reviews: 5</p>
+      </div>
+      <div class="product__price">
+        <p>Price:</p>
+        <p class="product__price-value">$2,199</p>
+      </div>
+      <a class="btn"
+      href="#"
+      data-qa="hover">
+      Buy
+      </a>
```

Comment:
```suggestion
      <a
        class="btn"
        href="#"
        data-qa="hover"
      >
        Buy
      </a>
```

Reply:
isn't fixed. apply code styling recommendation all over the page
https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules